src/http/definition.rs.in

```diff
@@ -105,7 +105,7 @@ fn find_definition_request_from_json() {
     assert_eq!(req.column, 3);
 }
 
-#[derive(Debug, Deserialize, Serialize)]
+#[derive(Debug, Serialize, Deserialize)]
```

Comment:
It doesn't actually matter. This was an attempt to fix a weird error that turned out to be due to an old version of serde_codegen. I'll revert this line.